ibmdbpy/frame.py

```diff
@@ -227,6 +227,15 @@ def indexer(self):
         else:
             None
 
+    #### added ####
+    def set_indexer(self, value):
+        """
+        Convenience function to define an indexer on an existing dataframe
```

Comment:
This is the same as https://github.com/ibmdbanalytics/ibmdbpy/blob/c5750cf8961fc0555af8bfb62c65116f1c5c1c0b/ibmdbpy/frame.py#L239-L259, which can be used like `idadf.indexer = <value>` (it's a [property](https://docs.python.org/3/library/functions.html#property)).

Reply:
well it doesn't work, the value is not updated with `idadf.indexer = <value>`, I don't know precisely why

Reply:
Yes, you are :-) Have a look at the property documentation I linked to (emphasis mine):

> If c is an instance of C, c.x will invoke the getter, _c.x = value will invoke the setter_ and del c.x the deleter.

You need to use

```python
users = IdaGeoDataFrame(idadb, "GEO_TOURISTS")
users.indexer = 'USER_ID'
```

Reply:
ok so I have deleted `set_indexer` from code and documentation